packages/react/src/components/Slider/Slider.tsx

```diff
@@ -84,10 +85,15 @@ export interface SliderProps
   inputType?: string;
 
   /**
-   * `true` to specify if the control is invalid.
+   * `Specify whether the Slider is currently invalid
    */
   invalid?: boolean;
 
+  /**
+   * Provide the text that is displayed when the Slider is in an invalid state
+   */
+  invalidText: React.ReactNode;
```

Comment:
should this one be optional as well? 
```suggestion
  invalidText?: React.ReactNode;
```